charts/trust-policies/Chart.yaml

```diff
@@ -8,8 +8,8 @@ sources:
 type: application
 
 name: trust-policies
-version: "v0.5.0"
-appVersion: "v0.5.0"
+version: "v0.6.0"
```

Comment:
Minor / just to make sure I understand: we're making this tag after this pull request lands?

Reply:
That's right!